client/components/app-promo/index.jsx

```diff
@@ -2,20 +2,22 @@
  * External dependencies
  */
 import React from 'react';
+import { connect } from 'react-redux';
 import Gridicon from 'components/gridicon';
 
-import analytics from 'lib/analytics';
+import { recordTracksEvent } from 'state/analytics/actions';
 import store from 'store';
 import userUtils from 'lib/user/utils';
+import { isChromeOS } from 'lib/user-agent-utils';
 import viewport from 'lib/viewport';
+import { translate } from 'i18n-calypso';
+import { getRandomPromo, getPromoLink } from './lib/promo-retriever';
 
-export default React.createClass( {
+const AppPromo = React.createClass( {
```

Comment:
if we `export const AppPromo = …` then we can also have our named export easily accessible